Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -2104,3 +2104,20 @@
     whitelist:
     - Chef
     - Mime
+
+- type: listing
+  id: UplinkCluwneBombs
+  name: uplink-cluwne-bombs-name
+  description: uplink-cluwne-bombs-desc
+  icon: { sprite:  Objects/Weapons/Bombs/cluwne_bomb.rsi, state: icon}
+  productEntity: BriefcaseSyndieCluwneBombBundleFilled
+  cost:
+    Telecrystal: 6
+  categories:
+  - UplinkJob
+  conditions:
+  - !type:BuyerJobCondition
+    whitelist:
+    - Clown
+    - Mime
```

Comment:
I'm not sure that the Mime should have access to this.

I don't think it meshes well with their silent theme to have a uplink item that is made to be highly obnoxious (unless the goal of giving access is to intentionally subvert it).

Reply:
yeah its not the most like, direct fitting, but i think it goes along with the vibes of them being rivals, since yeah it may incriminate them

same reason why passengers have it, it makes it harder to metagame it and i also dont likey to gatekeep syndi stuff too much

Reply:
nevermind it is there it just doesnt wanna show up